packages/manager/apps/container/src/core/container/ContainerProvider.jsx

```diff
@@ -21,27 +20,33 @@ export const ContainerProvider = ({ children }) => {
   // user choice about using or not the beta
   const [useBeta, setUseBeta] = useState(false);
 
+  // is the chatbot enabled ?
+  const [hasChatbot, setHasChatbot] = useState(false);
```

Comment:
Maybe we should rename this value instead of having a comment explaining what this is

```
const [isChatbotEnabled, setIsChatbotEnabled] = useState(false);
```